kratos/tests/test_model_part_io.py

```diff
@@ -5,10 +5,19 @@
 
 import KratosMultiphysics.kratos_utilities as kratos_utils
 
+try:
+    import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication

```

Comment:
Maybe this is already done, but I wonder if we should allow tests in core that depend on applications. Is there a reason not to put such tests in the application's tests?

Reply:
This is already done in other places
Mostly if e.g. Elements or ConstLaws are needed, but those are not actually implemented in the Core
I am not saying that it is a good solution though ...